packages/language-server/src/MessageHandler.ts

```diff
@@ -132,7 +134,12 @@ export function getBlockAtPosition(
     if (!reachedLine) {
       continue
     }
-    if (item.includes('{')) {
+    if (
+      allowedBlockIdentifiers.some((identifier) =>
```

Comment:
```suggestion
      allowedBlockIdentifiers.some(item.startsWith)
```

Reply:
Looks good! Can you please create a new PR for this so we can merge it?